tests/jobsautoreport/test_report.py

```diff
@@ -203,6 +203,102 @@
         variant="test",
         context="edge-e2e-metal-assisted",
     ),
+    JobDetails(
+        build_id="test",
+        duration=2053,
+        name="assisted-service-master-edge-metal-assisted-50",
+        refs=JobRefs(base_ref="master", org="openshift", repo="assisted-service"),
+        start_time=datetime.now() - timedelta(hours=1),
+        state="success",
+        type="postsubmit",
+        url="test",
+        variant="test",
+        context="edge-e2e-metal-assisted",
```

Comment:
let's do something more simulating for the postsubmit jobs to make it less confusing (again, all/most of the postsubmit jobs are not e2e/subsystem)

Reply:
Do you mean to edit the confusing parameters?

Reply:
change ``edge-e2e-metal-assisted`` to a real postsubmit job of ours